sql/ShardSet.go

```diff
@@ -17,6 +17,7 @@ type ShardSet struct {
 	password        string
 	host            string
 	port            int
+	tls             string
```

Comment:
should this be a string? why not a bool?

Reply:
Explained in the other comment thread.